src/cq_cam/fluent.py

```diff
@@ -45,18 +53,28 @@ def __init__(
         self,
         top: cq.Plane,
         feed: float,
+        speed: float,
```

Comment:
Speed should be left optional, as lower end CNC machines often can't control it anyway (like mine doesn't, I got an external VFD. Starting, stopping and configuring RPM is requires manual action on the VFD.)